### DISCUSSION TITLE
RPC Client/Server

### DISCUSSION THREAD

**Discussion by BetterCallBene** [Atostek/RustDDS#250] [Q&A]
Hello guys, is there a example for rpc server/client implementation?

Best regards

Benedikt

**Comment by jhelovuo** [comment]
Are you looking for DDS-RPC or ROS2 Service call?

For DDS-RPC, not at the moment.

For ROS2 Service calls, please see [ros2-client examples](https://github.com/jhelovuo/ros2-client/tree/master/examples) . The implementation for ROS2 Services is DDS-RPC.

**Reply by BetterCallBene** [reply]
Thanks for the quick response. I'm searching for general dds rpc calls because we are using rti connext. But I will have look how it is implemented in ros - hopefully I can adapt it.
